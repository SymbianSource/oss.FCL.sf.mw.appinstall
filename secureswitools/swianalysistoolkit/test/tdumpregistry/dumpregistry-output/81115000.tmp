Registry Controller Information:

Registry Entry :.\data\81115000\00000000.reg

Package UID :0x81115000
Package Name :053177c_0
Vendor :Unique Vendor Name
Package Index :0

Controller1 Version :1.2.3
Controller1 Algorithm :SHA1

Number of Dependencies : 0

Number of Embedded Packages : 0

SISProperty Count:0

Number Of File Descriptions :1

SIS File Description:

Target1:c:\Documents\def053177\appcap1.txt
File Operation Type :Install
Algorithm :SHA1
SID :0x0

Validation Status : ValidatedToAnchor
Revocation Status : OcspNotPerformed
