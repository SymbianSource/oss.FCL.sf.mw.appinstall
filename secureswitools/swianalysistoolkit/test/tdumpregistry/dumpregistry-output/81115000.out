Package Information:

Registry File :.\data\81115000\00000000.reg

Package UID :0x81115000
Package Name :053177c_0
Vendor :Unique Vendor Name
Package Index :0
Drives :4
No of executables :0
Package Version :1.2.3
Languages :English
Selected Drive :C
File Major Version :4
File Minor Version :0
Package Vendor Localized Name :Vendor
Package Install Type :Installation
Is Package in ROM :No
Package Preinstalled :No
Package Signed :Yes
Package Trust :SisPackageCertificateChainValidatedToTrustAnchor
